templates/taxi/driver_license_update_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
```

Comment:
This template is unnecessary - driver_form is handling both create and update

Reply:
ok, so i extends driver_form instead base.html?

Reply:
No, you actually can delete this template and reassign template_name on DriverLicenseUpdateView